index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" media="screen and (min-width: 768px)" href="./css/style-desktop.css" />
+    <title>Space Tourism Conference</title>
+</head>
+<body>
+    <header></header>
```

Comment:
- [ ] Please do not leave the `header` tag empty. I think you can use it wrap the `nav` instead